packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -90,6 +106,27 @@ const Columns = ({
               ? instructionsLabel
               : disabledInstructionsLabel}
           </span>
+          <div
+            id={'customize-columns-select-all'}
+            className={classNames(`${blockClass}__draggable-handleHolder`, {
+              ['-selected']: getVisibleColumnsCount() > 0,
+            })}
+            selected={getVisibleColumnsCount() > 0}
+          >
+            <Checkbox
+              wrapperClassName={`${blockClass}__customize-columns-checkbox-wrapper`}
+              className={`${blockClass}__customize-columns-select-all`}
+              checked={getVisibleColumnsCount() == columns.length}
```

Comment:
In general, when doing this kind of comparison, use `===` instead. `==` performs type conversion, `===` is a strict comparison which takes data type into account.

You may try the following experiment:
- 0 == false (evaluates to true)
- 0 === false (evaluates to false)

Reply:
Replaced `==` with `===`